styles/about.css

```diff
@@ -0,0 +1,71 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@700&display=swap');
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+} 
+
+.home {
+  font-weight: bold;
+}
+
+header {
+  background-color: #f9d199;
+  max-width: 100%;
+  height: 84px; 
+}
+
+.logo {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 20px;
+  line-height: 24px;
+  color: #000;
+  margin-top: 30px;
+  margin-left: 70px; 
+}
+
+.menu-header {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 20px;
+  line-height: 24px;
+  color: #000;
+  justify-content: space-between;
+  align-items: center;
+  display: flex;
+}
+
+.menu-header ul {
+  list-style: none;
+}
+
+.menu-header ul li {
+  display: inline;
+}
+
+ul {
+  margin-top: 30px;
+}
+
+.home {
+  margin-right: 80px;
+}
+
+.about {
+  margin-right: 80px;
+}
+
+.contact {
+  margin-right: 70px;
+}
```

Comment:
These styles seem too familiar, how about we simplified it:

`.navigator li {
  margin-right: 80px;
}`

Reply:
You just to simplify by reducing the number of lines?

Reply:
Yep. If you replace .home, .about and .contact to my code, It will work like the same way. Try it. The code will be simpler.